mne_features/tests/test_bivariate.py

```diff
@@ -13,8 +13,8 @@
                                     compute_time_corr_coefs)
 
 rng = np.random.RandomState(42)
-sfreq = 256.
-data = rng.standard_normal((10, 20, int(sfreq)))
+sfreq = 64.
+data = rng.standard_normal((3, 5, int(sfreq)))
```

Comment:
@jbschiratti with such smaller data I get segfault with numba.

Reply:
Yes, it seems there is an issue with `compute_nonlinear_interdep`. I'll fix that.